### ISSUE TITLE
Forms/Validation / error states - Helping Users Correct Errors

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#19]
**Requirement**: guidance should be presented to a user where there is an error, provided there is no risk to the security of the page or content.

**Recommendation**: Provide a suitable explanatory message explaining the error.
Examples:
- If the user incorrectly enters ‘18’ in a field labelled 'Number of widgets'. Instead of a message that does not provide guidance such as 'Invalid value', a suitable message would be ‘Must be between 1 and 10’.
- If the user enters ‘I do not have an email address’ in a field labelled ‘Email address’. A suitable message would be ‘Not a valid email address’.
- An example of not risking security would be not suggesting a correction for a mistake in a password field.
